client/state/initial-state.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import debugModule from 'debug';
+import localforage from 'localforage';
+
+/**
+ * Internal dependencies
+ */
+import { createReduxStore, reducer } from 'state';
+import { SERIALIZE, DESERIALIZE } from 'state/action-types'
+import config from 'config';
+
+/**
+ * Module variables
+ */
+const debug = debugModule( 'calypso:state' );
+
+export const localforageConfig = {
+	name: 'calypso',
+	storeName: 'calypso_store',
+	description: 'Calypso Storage',
+	driver: localforage.INDEXDB
```

Comment:
Actually, I don't need to force the driver anymore since we're not using msgpack!